config/console.php

```diff
@@ -16,4 +18,6 @@
         'createPath()' => [$params['yiisoft/yii-db-migration']['createPath']],
         'updatePath()' => [$params['yiisoft/yii-db-migration']['updatePath']],
     ],
+
+    MigrationInformerInterface::class => NullMigrationInformer::class,
```

Comment:
Should it be `ConsoleMigrationInformer` by default?

Reply:
I think no.
For example I want use migrator programmaticaly - inject via container and messages will be output to console - is bad.

If migrator use from console, then console command should set `ConsoleMigrationInformer`.